bundles/org.openhab.ui.basic/web-src/smarthome.js

```diff
@@ -1361,6 +1392,7 @@
 				}
 			});
 			componentHandler.downgradeElements(_t.buttons);
+			window.removeEventListener("resize", _t.minimizeWidth);
```

Comment:
Maybe remove it only if it was added first ?
That would mean adding the test on `_t.buttons.length > 1 && _t.parentNode.classList.contains(o.buttonsMultilineClass)`

Reply:
Is there any harm in removing it like this? If _t.minimizeWidth is null, [removeEventListener()](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener) doesn't mind.

> Calling removeEventListener() with arguments that do not identify any currently registered [event listener](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#the_event_listener_callback) on the EventTarget has no effect.

Reply:
Ok, thank for the link.